Standards/scs-0302-v1-domain-manager-role.md

```diff
@@ -89,7 +89,7 @@ OpenStack Keystone allows for new roles to be created via its API by administrat
 Additionally, each OpenStack API's RBAC can be adjusted through an API policy file (`policy.yaml`) through olso-policy[^4], Keystone included.
```

Comment:
The headline should be changed to persona, or something alike (a "role model")
#### Introducing a new role and API policy changes